cmd/bosun/sched/check.go

```diff
@@ -174,6 +174,13 @@ func (s *Schedule) RunHistory(r *RunHistory) {
 		// If the old alert was not acknowledged, do nothing.
 		// Do nothing if state did not change.
 		notify := func(ns *conf.Notifications) {
+			if a.Log {
+				lastLogTime, ok := s.lastLogTimes[ak]
+				if ok && lastLogTime.Add(a.MaxLogFrequency).After(time.Now()) {
```

Comment:
LastTime + 5m is next time we can log. 

If now is before that we should not log. I think mine is correct but I agree it would make more sense to do `now.Before(last + duration)`